Docs/source/running_cpp/parameters.rst

```diff
@@ -45,7 +45,7 @@ Overall simulation parameters
     printed to standard output. Currently only works if the Lorentz boost and
     the moving window are along the z direction.
 
-* ``warpx.verbose`` (`0` or `1`)
+* ``warpx.verbose`` (``0`` or ``1``; default is ``0`` for false)
```

Comment:
Is the default not set in:
https://github.com/ECP-WarpX/WarpX/blob/89b4801d778f58f04162b2c886c6c57ea4cae009/Source/WarpX.H#L775

Reply:
Thanks for catching this!
```suggestion
* ``warpx.verbose`` (``0`` or ``1``; default is ``1`` for true)
```